app/src/components/settings/UserForm.tsx

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import { Controller, useForm } from "react-hook-form";
+import { css } from "@emotion/react";
+
+import { Button, Flex, Form, TextField, View } from "@arizeai/components";
+
+import { UserRole } from "@phoenix/constants";
+
+import { RolePicker } from "./RolePicker";
+
+export type UserFormParams = {
+  email: string;
+  username: string | null;
+  password: string;
+  role: UserRole;
+};
+
+export function UserForm({
+  onSubmit,
+  email,
+  username,
+  password,
+  role,
+  isSubmitting,
+}: {
+  onSubmit: (data: UserFormParams) => void;
+  isSubmitting: boolean;
+} & Partial<UserFormParams>) {
+  const {
+    control,
+    handleSubmit,
+    formState: { isDirty },
+  } = useForm<UserFormParams>({
+    defaultValues: {
+      email: email ?? "",
+      username: username ?? null,
+      password: password ?? "",
+      role: role ?? UserRole.ADMIN,
+    },
+  });
+  return (
+    <div
+      css={css`
+        .role-picker {
+          width: 100%;
+          .ac-dropdown--picker,
+          .ac-dropdown-button {
+            width: 100%;
+          }
+        }
+      `}
+    >
+      <Form onSubmit={handleSubmit(onSubmit)}>
+        <View padding="size-200">
+          <Controller
+            name="email"
+            control={control}
+            rules={{
+              required: "Email is required",
+              pattern: {
+                value: /^[^@\s]+@[^@\s]+[.][^@\s]+$/,
+                message: "Entered value does not match email format",
+              },
+            }}
+            render={({
+              field: { name, onChange, onBlur, value },
+              fieldState: { invalid, error },
+            }) => (
+              <TextField
+                label="Email"
+                type="email"
+                name={name}
+                errorMessage={error?.message}
+                validationState={invalid ? "invalid" : "valid"}
+                onChange={onChange}
+                onBlur={onBlur}
+                value={value.toString()}
```

Comment:
does just value work here?
```suggestion
                value={value}
```